cypress.json

```diff
@@ -1,8 +1,10 @@
 {
-  "baseUrl": "http://localhost:9000/",
+  "baseUrl": "http://localhost:8000/",
   "integrationFolder": "cypress/integration",
   "defaultCommandTimeout": 20000,
   "video": true,
   "retries": 2,
-  "chromeWebSecurity": false
+  "chromeWebSecurity": false,
+  "viewportWidth": 1000,
+  "viewportHeight": 600
```

Comment:
Does this change is essential?!

Reply:
@heitorado Do you remember?

Reply:
I wouldn't say they're essential since these are the default values, but it can be helpful to have them specified so other people can easily spot where to change them.

I think the idea here was to specify these values so they match with what is [currently used on the CI](https://github.com/vtex/storeframework-platform/blob/7b3968a623a2a7b3915a458c02497c8390587950/control_plane/images/cypress-mochawesome/src/constants/CypressConfig.ts#L41-L42).